js/webpack_get_defaults.js

```diff
@@ -8,6 +8,7 @@ const path = require('path')
 const HtmlPlugin = require('html-webpack-plugin')
 const ExtractTextPlugin = require('extract-text-webpack-plugin')
 const UglifyPlugin = require('uglifyjs-webpack-plugin')
+const Webpack = require('webpack')
```

Comment:
nitpick: généralement webpack est importé en minuscules dans tous les plugins de l'ecosysteme et des exemples du site officiel.

Reply:
ok I'll fix it.